### DISCUSSION TITLE
What is your experience with the models from "rtdl"?

### DISCUSSION THREAD

**Discussion by Yura52** [yandex-research/rtdl#1] [Announcements]
Hi! Thank you for your interest in `rtdl`! We suggest sharing feedback on the **models** from the `rtdl` library in this discussion. This discussion can help users make decisions whether they should try some model(s) in their tasks.

You can cover _anything_ you think is important. For example:
- have you successfully applied some model from `rtdl` to solve some task?
- what was the task type? What was the dataset size? What was the number of features? What was the domain?
- what models did you compare? Did you compare them with Gradient Boostings? What was the result in terms of target metrics and other aspects? (efficiency, API, etc.)
- if you tried to use some model from `rtdl`, but then decided to use other solution, what were the reasons?
- and anything else!

**Comment by thelexhibition** [comment]
Hi, I am interested in trying the algorithm but noticed the example in Colab only has an example for how it is used in a regression task and it is not clear to me how to modify it to work for classification. Is it possible to provide a simple example of it used in classification for the Forest Cover Type dataset for example? Thanks

**Reply by Yura52** [reply]
We have extended the notebook with a classification example. The only thing you need is to uncomment the corresponding code in the cell where data is prepared. Note that the classification dataset that we added is quite big and you will need GPU/TPU in order to train FT-Transformer in a reasonable time. However, CPU should be enough for MLP/ResNet.
